orc8r/cloud/docker/run.py

```diff
@@ -50,6 +50,16 @@ def main() -> None:
     if args.print:
         return
 
+    if args.clear_db:
+        try:
+            cmd = ['docker-compose', 'down']
+            subprocess.run(cmd, check=True)
+            cmd = ['docker', 'volume', 'rm', '-f', 'orc8r_pgdata']
```

Comment:
Nit here and elsewhere: default to long-form of flags, since it's self-documenting